docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+* Remove support for delaying state recovery pending master {es-pull}53845[#53845] (issue: {es-issue}51806[#51806])
+
+Distributed::
+* Remove synced flush {es-pull}50882[#50882] (issues: {es-issue}50776[#50776], {es-issue}50835[#50835])
+* Remove the `cluster.remote.connect` setting {es-pull}54175[#54175] (issue: {es-issue}53924[#53924])
+
+Engine::
+* Force merge should reject requests with `only_expunge_deletes` and `max_num_segments` set {es-pull}44761[#44761] (issue: {es-issue}43102[#43102])
+* Remove per-type indexing stats {es-pull}47203[#47203] (issue: {es-issue}41059[#41059])
+* Remove translog retention settings {es-pull}51697[#51697] (issue: {es-issue}50775[#50775])
+
+Features/CAT APIs::
+* Remove the deprecated `local` parameter for `_cat/indices` {es-pull}64868[#64868] (issue: {es-issue}62198[#62198])
+* Remove the deprecated `local` parameter for `_cat/shards` {es-pull}64867[#64867] (issue: {es-issue}62197[#62197])
+
+Features/Features::
+* Remove deprecated ._tier allocation filtering settings {es-pull}73074[#73074] (issue: {es-issue}72835[#72835])
+
+Features/ILM+SLM::
+* Add lower bound on `poll_interval` {es-pull}39593[#39593] (issue: {es-issue}39163[#39163])
+* Make the ILM `freeze` action a no-op {es-pull}77158[#77158] (issue: {es-issue}70192[#70192])
+* Always enforce default tier preference {es-pull}79751[#79751] (issue: {es-issue}76147[#76147])
+* Validate that snapshot repository exists for ILM policies at creation/update time {es-pull}78468[#78468] (issues: {es-issue}72957[#72957], {es-issue}77657[#77657])
+* Default `ENFORCE_DEFAULT_TIER_PREFERENCE` to `true` {es-pull}79275[#79275] (issues: {es-issue}76147[#76147], {es-issue}79210[#79210])
+
+Features/Indices APIs::
+* Change prefer_v2_templates parameter to default to true {es-pull}55489[#55489] (issues: {es-issue}53101[#53101], {es-issue}55411[#55411])
+* Remove deprecated `_upgrade` API {es-pull}64732[#64732] (issue: {es-issue}21337[#21337])
+* Remove local parameter for get field mapping request {es-pull}55100[#55100] (issue: {es-issue}55099[#55099])
+* Remove `include_type_name` parameter from REST layer {es-pull}48632[#48632] (issue: {es-issue}41059[#41059])
+* Remove the `template` field in index templates {es-pull}49460[#49460] (issue: {es-issue}21009[#21009])
+* Remove endpoint for freezing indices {es-pull}78918[#78918] (issues: {es-issue}70192[#70192], {es-issue}77273[#77273])
+
+Features/Watcher::
+* Move watcher history to data stream {es-pull}64252[#64252]
+
+Geo::
+* Disallow creating `geo_shape` mappings with deprecated parameters {es-pull}70850[#70850] (issue: {es-issue}32039[#32039])
+* Remove bounding box query `type` parameter {es-pull}74536[#74536]
+
+Infra/Circuit Breakers::
+* Fixed synchronizing inflight breaker with internal variable {es-pull}40878[#40878]
+
+Infra/Core::
+* Fail when using multiple data paths {es-pull}72184[#72184] (issue: {es-issue}71205[#71205])
```

Comment:
We reverted this in #79116 as part of reverting the removal of multiple data paths. See #78525.

Reply:
Removed this item with https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c